frameworks/hdfs/universe/config.json

```diff
@@ -14,6 +14,11 @@
           "description":"The principal for the HDFS service instance.",
           "type":"string",
           "default":"hdfs-principal"
+        },
+        "cmd_prefix" : {
```

Comment:
Yeah, I don't want this here, but as you say, I have nowhere else to put it.  I think it's better than in Marathon.json.mustache which was getting to be a huge ugly command.  Also the Template framework will just get this, things will stay a bit neater.